packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -641,6 +659,7 @@ class FlutterPlatform extends PlatformPlugin {
     return generateTestBootstrap(
       testUrl: testUrl,
       testConfigFile: findTestConfigFile(globals.fs.file(testUrl), globals.logger),
+      packageConfigUri: buildInfo != null ? globals.fs.path.toUri(buildInfo!.packagesPath) : null,
```

Comment:
is a relative URI here ok?

Reply:
No, this needs to be a file URI. This is documented in the `generateTestBootstrap` method dart doc

Reply:
I'm saying this *will* be a relative uri, since I believe `buildInfo!.packagesPath` is relative. Is this ok?

Reply:
the `globals.fs.path.toUri` method takes the path and returns a file URI so `generateTestBootstrap` will still get the file URI it needs.